makers/senior_engineer.md

```diff
@@ -1,35 +1,76 @@
+
+- Take the lead in designing and implementing solutions to moderately-complex, loosely-scoped problems.
+- Deliver large features that touch multiple product domains, working independently or leading other engineers.
+- Identify risks, tradeoffs, and alternative solutions before making design decisions.
+- Keep stakeholders informed of likely delays, and suggest workarounds for complications.
+- Spearhead investigations into hotfixes and other bugs, regardless of origin.
+- Suggest improvements to the team's technology and tools based on industry trends and best practices.
+- Identify tech debt before it becomes a crisis. Work with the PM and other senior engineers to prioritize it appropriately.
+
+[Example Behaviors](/README.md#example-behaviors):
+- Took ownership of debugging, writing, and deploying a hotfix
+- Reduced the scope of a feature to meet a deadline
+- Divided a large feature into smaller tasks that could be completed in parallel
+- Refactored a module to be more efficient before adding additional functionality
+
+---
+### [Business Knowledge](/README.md#business-knowledge)
+
+- Become an expert in multiple areas of the codebase or product.
+- Consider team goals, product direction, and company objectives when designing technical solutions.
+
+[Example Behaviors](/README.md#example-behaviors):
+- TBA
+
+---
+### [Communication](/README.md#communication)
+
+- Facilitate productive meetings with clear outcomes.
+- Encourage inclusive discussions and decision-making by mindfully inviting participation from less-heard voices.
+- Serve as a team representative in cross-team meetings and initiatives.
+- Solicit buy-in when making technical decisions.
+- Avoid silos by actively sharing information with your team about best practices, technical approaches, etc.
+- Communicate technical concepts clearly, regardless of the technical familiarity of your audience.
+- Articulate and advocate for the product team's goals to engineers on your team.
+
+[Example Behaviors](/README.md#example-behaviors):
+- Gave a presentation about DataDog to other developers at a guild meeting
+- Explained a technical issue in an understandable way to a non-technical stakeholder
+- Created a diagram to document how different parts of system interact
+- Answered another engineer's question about the business value of a particular piece of work
+
+---
+### [Leadership & Teamwork](#leadership--teamwork)
+
+- Set and maintain high expectations for yourself and others.
+- Nurture an empathetic, no-blame culture on your team.
+- Prioritize work that unblocks your teammates.
+- Provide constructive, actionable feedback to your colleagues in an empathetic way.
+- Suggest solutions to problems you notice with our product and processes.
+- Notice if morale is low, and work with the team to identify and address the causes.
+- Coach junior engineers via pairing and code reviews.
+- Volunteer to serve as a formal mentor for junior engineers.
```

Comment:
Do we want to use "junior" here or should we use verbiage that reflects the updates we made to the ladder? (Engineer I and II). Not sure what the best phrasing would be in the latter.

"Coach engineers starting their career via pairing and code reviews."?